src/components/BookList.js

```diff
@@ -1,15 +1,29 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 import Book from './Book';
 
 const BookList = () => {
-  const books = [
-    { title: 'Mindset', author: 'Shadman', id: 0 },
-    { title: 'Eat that frog', author: 'Naresh', id: 1 },
-  ];
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = (id) => {
+    dispatch(removeBook(id));
+  };
 
   return (
     <div>
       {books.map((book) => (
-        <Book key={book.id} title={book.title} author={book.author} />
+        <div key={book.item_id}>
+          <Book
+            title={book.title}
+            author={book.author}
+            category={book.category}
+          />
+          <button type="button" onClick={() => handleRemoveBook(book.item_id)}>
+            Remove Book
+          </button>
+        </div>
       ))}
     </div>
```

Comment:
- [ ] I suggest that you move these `jsx` codes to the `<Book>` component then `books` are accessed from inside the `<BookList>` with `useSelector`